Arena/ArenaHooks.cs

```diff
@@ -1118,8 +1118,13 @@ private void HUD_InitMultiplayerHud(On.HUD.HUD.orig_InitMultiplayerHud orig, HUD
 
             if (isArenaMode(out var arena))
             {
+                self.AddPart(new TextPrompt(self));
```

Comment:
while this is a draft, just wanted to let you know to shove the HUD components inside line 1127 so all game modes would inherit. 

https://github.com/henpemaz/Rain-Meadow/blob/main/Arena/ArenaOnlineGameModes/BaseGameMode.cs#L64

And to remove pointing from the added HUD components since it's not synced.